src/cert_context.rs

```diff
@@ -23,6 +24,20 @@ pub struct CertContext(winapi::PCCERT_CONTEXT);
 unsafe impl Sync for CertContext {}
 unsafe impl Send for CertContext {}
 
+// TODO: unsure if this is a solid decision instead of a fn `matches`
+// should work anyways
+impl cmp::PartialEq for CertContext {
+    fn eq(&self, other: &CertContext) -> bool {
+        let ret = unsafe {
+            crypt32::CertComparePublicKeyInfo(
```

Comment:
@sfackler 
Yeah I first thought about simply using `.sha1` but after SHAttered I prefered SHA256, which isn't easily accessible yet, so I settled on just comparing the public key for now.
I'll compare SHA256 hashes in the next draft.